web/src/stream_list.ts

```diff
@@ -851,7 +851,8 @@ export function initialize({
 }
 
 export function initialize_tippy_tooltips(): void {
-    tippy.default("#stream_filters li .subscription_block .stream-name", {
+    tippy.delegate("#stream_filters", {
```

Comment:
We should just attact this to body to avoid the race with `#stream_filters` being attached to the DOM vs with this event being attached.

Reply:
Updated to that, thanks!